### DISCUSSION TITLE
possible to individually set /player render distance?

### DISCUSSION THREAD

**Discussion by natopotato390** [gnembon/fabric-carpet#1393] [Ideas]
Wondering if this feature could even be possible, could you be able to set a specific render distance size for spawned in /players?
This would be extremely helpful in certain situations, for example:
Your server render distance is something high 10-16ish, say you want to send some end ice highway machine pretty far.
being able to set the bot's render distance to 2 would reduce the world file size immensely without having to set the server render distance to 2 for everyone on.

**Comment by Coderx-Gamer** [comment]
It should be possible but might have some weird bugs, also it would not be vanilla friendly If u made some contraption that will rely on it.
